string tether_length_string
---
bool success
